docs/tutorial_step1.md

```diff
@@ -67,6 +67,9 @@ Now run Sugarizer, you should see the icon of a new activity like this:
 
 ![](images/tutorial_step1_2.png)
 
+Note: If you are not able to see the icon of newly added activity, empty the browser cache and reload
```

Comment:
Not sure why it's needed.

Reply:
In some browser(in my case Safari) if you don't hard reload(removing cache) it doesn't reflect new changes. I myself encounter this error, and thought that small note might help other beginners.